encrypt-config/ArgParser.h

```diff
@@ -48,20 +48,20 @@ class Arguments {
 
   void set(const std::string& bool_key);
 
-  static utils::optional<Argument> getArg(const std::string& key);
-  static utils::optional<Flag> getFlag(const std::string& flag);
+  static std::optional<Argument> getArg(const std::string& key);
+  static std::optional<Flag> getFlag(const std::string& flag);
 
  public:
   static Arguments parse(int argc, char* argv[]);
 
   static std::string getHelp();
 
-  utils::optional<std::string> get(const std::string& key) const;
+  [[nodiscard]] std::optional<std::string> get(const std::string& key) const;
```

Comment:
I added a bunch of these `[[nodiscard]]` attributes where my IDE showed clang-tidy suggestions to do so.